react-app/src/components/ActionForm/ActionForm.test.tsx

```diff
@@ -290,6 +294,61 @@ describe("ActionForm", () => {
     });
   });
 
+  test("sends a cusomt GA event", async () => {
```

Comment:
```suggestion
  test("sends a custom Google Analytics event", async () => {
```

I don't know what's GA without this PR's context